### ISSUE TITLE
relative-url-prefix go parent issue

### ISSUE THREAD

**Issue by Delagen** [angular-eslint/angular-eslint#334]
Why rule does not allow double parent scheme as for example?
```
styleUrls:[ `../../file.scss`]
```
Due regexp I see that it only accept single level
```
const RELATIVE_URL_PREFIX_MATCHER = /^\.{1,2}\/[^./]/;
```

**Comment by JamesHenry** [comment]
@rafaelss95 Didn't we update the relative url rule recently?

**Comment by rafaelss95** [comment]
@JamesHenry Yes, we merged the PR #224, which fixed the problem described in #200, _it is_, the support for `../`. Now, to support deeper levels, we could update the `RegExp` to something like `/^\.\.?\/.+/`. Do you think that would still be in accordance with the Angular style guide?
